src/proteus/dswx_hls.py

```diff
@@ -3479,7 +3479,7 @@ def _populate_dswx_metadata_processing_parameters(
 
     # Copernicus class for forest masking
     dswx_metadata_dict['FOREST_MASK_LANDCOVER_CLASSES'] = \
-        ', '.join(forest_mask_landcover_classes)
+        ','.join([str(c) for c in forest_mask_landcover_classes])
 
     # ocean masking distance from shoreline in km
     dswx_metadata_dict['OCEAN_MASKING_SHORELINE_DISTANCE_KM'] = \
```

Comment:
Should this only be a number if a shoreline shape file is provided?

If there's no shoreline file, then we do not create an ocean mask, correct? In which case, should this be marked with an e.g. `-`, similar to how `MIN_SLOPE_ANGLE` is handled depending on `SHADOW_MASKING_ALGORITHM` used?

Reply:
Uhm, `FOREST_MASK_LANDCOVER_CLASSES` has nothing to do with ocean masking.

Reply:
Ah - my apologies - I highlighted the incorrect lines.

Currently, for the product metadata, the block of code on lines 3484-3485 will always set `OCEAN_MASKING_SHORELINE_DISTANCE_KM` to `ocean_masking_shoreline_distance_km`. But, what happens if there is no shoreline shape file provided? Is it correct to still set the metadata field `OCEAN_MASKING_SHORELINE_DISTANCE_KM` to the default number, e.g. `1` for 1 km?

Or, if there is no shoreline shapefile provided, would it be better to set `OCEAN_MASKING_SHORELINE_DISTANCE_KM` to `-`? This approach would be similar to how the code is handling `SHADOW_MASKING_ALGORITHM` when the alternate algorithm is used ([link](https://github.com/gshiroma/PROTEUS/blob/32d1a1315b4ecb2886267f013d1ccea1d956c412/src/proteus/dswx_hls.py#L3527-L3528)).

Reply:
Sure, sounds good, @nemo794 . I updated the code to assign `-` to the metadata field `OCEAN_MASKING_SHORELINE_DISTANCE_KM` if the shoreline shapefiile is not provided.